platform/features/timeline/src/actions/CompositionColumn.js

```diff
@@ -41,7 +42,9 @@ define([], function () {
     CompositionColumn.prototype.value = function (domainObject) {
         var model = domainObject.getModel(),
             composition = model.composition || [];
-        return (composition[this.index]) || "";
+
+        return composition.length > this.index ?
+            this.idMap[composition[this.index]] : "";
```

Comment:
I *think* this can be replaced with 
```javascript
this.idMap[composition[this.index]]
```
Because `this.idMap[undefined] === undefined`